cypress/integration/details_spec.js

```diff
@@ -0,0 +1,16 @@
+describe('Details', () => {
+  beforeEach(() => {
+    cy.load()
+    cy.get('.card').click();
+});
+
+  it('Should be able to return to main menu', () => {
+    cy.get('.card').should('not.exist')
+      .get('.details').should('be.visible')
+      .get('.details').contains('Money Plane')
+
+      .get('button').click()
+      .get('.card').should('be.visible')
+      .get('.details').should('not.exist')
+  })
+});
```

Comment:
I'm glad we figured out we had to use not.exist with react instead of not.be.visable because React doesn't render certain elements so they don't exist, and not.be.visable is checking for css properties such as display: none or visibility: hidden.